### PR TITLE
WIP 1.18 support

### PR DESCRIPTION
**Pull request by Combustible**

### CODE REVIEW

File: utility_code/prune_empty_regions.py [hunk 1]
```diff
@@ -32,17 +32,16 @@ def process_region(arg):
...
-            num_entities += chunk.nbt.count_multipath('Entities[]')
+            num_entities += chunk.nbt.count_multipath('Level.Entities[]')
 
             # Check if there are tile entities (fast check)
-            num_block_entities += chunk.nbt.count_multipath('Level.TileEntities[]')
-
-            # Check if there are block entities (fast check)
-            num_block_entities += chunk.nbt.count_multipath('Level.BlockEntities[]')
+            num_block_entities += chunk.nbt.count_multipath('Level.TileEntities[]') # 1.17 and before
+            num_block_entities += chunk.nbt.count_multipath('Level.BlockEntities[]') # Not sure this was ever a thing?
```
Comment (NickNackGus) [inline utility_code/prune_empty_regions.py:39]: I don't see it in the change logs for the chunk format, safe to remove this line.

**Review by NickNackGus — APPROVED** [review]
Approved, just give it a test run if you haven't already.

**Comment by Combustible** [comment]
Merged manually

File: utility_code/prune_empty_regions.py (hunk 1 above)
Comment (Combustible) [reply utility_code/prune_empty_regions.py:39]: Removed

**Pull request closed without merging**